src/xcode/ENA/ENA/Source/Services/Risk/Provider/Helper/CachedAppConfiguration.swift

```diff
@@ -138,6 +137,13 @@ final class CachedAppConfiguration {
 			} // eo async
 		}
 	}
+	
+	private func storeETagInClientMetadata(eTag: String?) {
+		guard let store = self.store as? Store else {
+			return
+		}
+		store.clientMetadata = ClientMetadata(etag: eTag)
```

Comment:
```suggestion
	store?.clientMetadata = ClientMetadata(etag: eTag)
```
Shouldn't this work, too?

Reply:
Actually no, because the store in this class is confirming to "AppConfigCaching" and we need to cast it to "Store" to be able to access the metadata property, maybe it is more clear the screenshots.
<img width="839" alt="Screenshot 2021-02-12 at 10 58 30" src="https://user-images.githubusercontent.com/15270737/107754543-eb106280-6d21-11eb-8f9a-0981778b6544.png">
<img width="587" alt="Screenshot 2021-02-12 at 10 57 51" src="https://user-images.githubusercontent.com/15270737/107754537-e9df3580-6d21-11eb-8dba-7148227462a0.png">

Reply:
there is already a store in the class, why create a new instence? @naveeddotio

Reply:
If you really, really need this here, you could extend the protocol conformance of the property to:
`private let store: AppConfigCaching & PrivacyPreservingProviding`. No need to use the fat `Store` here.

On the other hand we create new dependencies between this app configuration and ppa.

Reply:
@whiskey my idea was close to that, i didn't want the whole CachedAppConfiguration to access the whole protocols for PPa instead just use it in this ppa specific function, since the injected store is already of type "Store" in the initializer

Reply:
We/You could 'shrink' that Store also down then and keep this slim :)

Reply:
mmm i think i will extend the protocol conformance for now atleast :')